src/main/java/picard/arrays/ZCallPedFile.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2019 The Broad Institute
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package picard.arrays;
+
+import htsjdk.samtools.util.IOUtil;
+import picard.PicardException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents a single-sample .ped file output from ZCall.
+ */
+class ZCallPedFile {
+    private static final int OFFSET = 6;
+    private final Map<String, String> snpToAlleleMap = new HashMap<>();
+
+    private void addAllele(final String snp, final String allele) {
+        snpToAlleleMap.put(snp, allele);
+    }
+
+    String getAlleles(final String snp) {
+        return snpToAlleleMap.get(snp);
+    }
+
+    /**
+     *
+     * @param pedFile  .ped files are whitespace-separated files.
+     *                The .ped format is defined at http://zzz.bwh.harvard.edu/plink/data.shtml#ped.
+     *                The first six columns, in the following order, are mandatory:
+     *                  Family ID
+     *                  Individual ID
+     *                  Paternal ID
+     *                  Maternal ID
+     *                  Sex (1 = male, 2 = female, other = unknown)
+     *                  Phenotype
+     *                The seventh column onward should be biallelic genotype data. ZCall outputs these as A/B,
```

Comment:
I thought that the seventh column onwards are, in pairs the first and second allele meaning that A/B will be encoded as `\tA\tB`.

Reply:
also, we are expecting this file to have exactly one line, right?

Reply:
Ah, yes, I didn't actually put that in the comments or javadoc.

Reply:
Also, @yfarjoun, I believe that your first comment and my javadoc are equivalent statements. What do you see as the differences?

Reply:
I think that a naive user might understand your javadoc to mean that the bed file should be encoded thusly (here as an example with _one_ snp):
```
<FAM>\t<IND>\t<PAT>\t<MAT>\t<SEX>\t<PHEN>\t[AB]\\[AB] (that is, with a literal slash)
```
while it should be encoded as (still for one snp):
```
<FAM>\t<IND>\t<PAT>\t<MAT>\t<SEX>\t<PHEN>\t[AB]\t[AB] (that is, with the two single character alleles separated with a tab character)
```

Reply:
Gotcha, thanks for the clarification.